README.md

```diff
@@ -0,0 +1,41 @@
+# OpenShift Runbooks
+A collection of runbooks for alert rules that are shipped with OpenShift.
+
+## How To Use
+OpenShift Container Platform is shipping a finely tuned set of alerts to inform
+the cluster's owner and/or operator of events and bad conditions in the cluster.
+This repository is a consistently growing collection of runbooks for said
+alerts, that are intended to guide a cluster owner and/or operator through the
+steps of fixing problems on clusters, which are surfaced by alerts.
+
+Going forward alerts in OpenShift will ship a link to the corresponding runbook
+in this repository, to make fixing problems even easier.
+
+## Repository Layout
+Runbooks in this repository are grouped by the operator that is responsible for
```

Comment:
are there only operators? Or should it be "component"

Reply:
I specifically used "operator" so people don't add one folder per subcomponent. EG all monitoring things are under cluster-monitoring-operstor and not Prometheus operator and alertmanager and Prometheus